src/python/pants/backend/python/util_rules/interpreter_constraints.py

```diff
@@ -192,7 +192,7 @@ def minimum_python_version(self) -> str | None:
         """
         if self.includes_python2():
             return "2.7"
-        max_expected_py3_patch_version = 15  # The current max is 3.6.12.
+        max_expected_py3_patch_version = 15  # The current max is 3.6.13.
         for major_minor in ("3.5", "3.6", "3.7", "3.8", "3.9", "3.10"):
```

Comment:
Here and elsewhere, I think that our list of major+minor versions should become an option.